deployment/ansible/roles/raster-foundry.jenkins/tasks/main.yml

```diff
@@ -24,3 +29,14 @@
       -v /etc:/etc
       spotify/docker-gc
     state: "present"
+
+- name: Restart Jenkins every 3 hours
+  cron:
+    name: "jenkins-restart"
+    user: "{{ jenkins_name }}"
+    hour: "*/3"
```

Comment:
I've been checking Jenkins every 3 hours or so to see if any indexing job is hanging, and that seems like a number which is infrequent enough to not be constantly disturbing builds, but frequent enough to catch a hanging job when it happens.